constants/extraRpcs.js

```diff
@@ -3690,6 +3690,12 @@ export const extraRpcs = {
       },
     ]
   },
+  3068: {
+    rpcs: [
```

Comment:
these 2 rpcs are already on https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-3068.json, there's no need to add them again here as they'll be imported

Reply:
Recheck